src/css/styles.css

```diff
@@ -11,6 +11,8 @@
 
   --color-twitter-blue: #55acee;
   --color-meetup-red: #ed1c40;
+  --color-twitter-blue-hover: #4c9cd9;
+  --color-meetup-red-hover: #bc1431;
 
```

Comment:
I made up a meetup red hover.  Can check with designer for the real value. in the mock has typo and meetuphover is same as twitter.